packages/eas-cli/src/update/configure.ts

```diff
@@ -252,6 +254,71 @@ async function ensureEASUpdatesIsConfiguredNativelyAsync(
   }
 }
 
+/**
+ * Make sure EAS Build profiles are configured to work with EAS Update by adding channels to build profiles.
+ */
+
+async function ensureEASUpdateIsConfiguredInEasConfigAsync({
+  projectDir,
+}: {
+  projectDir: string;
+}): Promise<void> {
```

Comment:
IMO pass just `projectDir` to the function (not an object with `projectDir`). There's no need to make this future-proof as this is really easy to change.